packages/js/src/types.ts

```diff
@@ -99,6 +118,29 @@ export type Cta = {
   action?: MessageAction;
 };
 
+export type WorkflowInfo = {
+  _id: string;
+  name: string;
+  critical: boolean;
+  tags?: string[];
+  identifier: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data?: Record<string, any>;
+};
+
+export type ChannelPreference = {
+  email?: boolean;
+  sms?: boolean;
+  in_app?: boolean;
```

Comment:
```suggestion
  inApp?: boolean;
```

Reply:
this is due to internal implementation and will require changes across the stack :(